staging/src/k8s.io/apiserver/pkg/server/secure_serving.go

```diff
@@ -170,7 +170,7 @@ type NamedTLSCert struct {
 	Names []string
 }
 
-// getNamedCertificateMap returns a map of *tls.Certificate by name. It's is
+// getNamedCertificateMap returns a map of *tls.Certificate by name. It's
```

Comment:
Can you also capitalize the G i.e `GetNamedCertificateMap` instead of `getNamedCertificateMap`?

Reply:
sure.